tests/foreman/sys/test_rename.py

```diff
@@ -0,0 +1,128 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``katello-change-hostname``
```

Comment:
Hello, this is how the command is called, both upstream and downstream, something like katello-backup